gc/2020/GC-10-20.md

```diff
@@ -0,0 +1,251 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the October 20th video call of WebAssembly's Garbage Collection Subgroup
+
+- **Where**: zoom.us
+- **When**: October 20th, 4pm-5pm UTC (October 20th, 9am-10am Pacific Daylight Time)
+- **Location**: *link on calendar invite*
+
+### Registration
+
+None required if you've attended before. Fill out the form here to sign up if
+it's your first time: https://forms.gle/JehrAB4gWbtHjybt9. The meeting is open
+to CG members only.
+
+## Logistics
+
+The meeting will be on a zoom.us video conference.
+Installation is required, see the calendar invite.
+
+## Agenda items
+
+1. Opening, welcome and roll call
+    1. Opening of the meeting
+    1. Introduction of attendees
+1. Find volunteers for note taking (acting chair to volunteer)
+1. Adoption of the agenda
+1. Proposals and discussions
+    1. Discussion: Observations about Types in Wasm (Andreas Rossberg) [1 hr]
+1. Closure
+
+## Meeting Notes
+
+### Introduction of attendees
+
+Thomas Lively
+Wouter van Oortmerssen
+Francis McCabe
+Tim Steenvoorden
+Ben Titzer
+Slava Kuzmich
+Conrad Watt
+Zhi An Ng
+Daniel Wirtz
+Drew Youngwerth
+Ben Smith
+Rick Battagline
+Emanuel Ziegler
+Jakob Kummerow
+Tobias Tebbi
+Andreas Rossberg
+Ross Tate
+Zalim Bashorov
+Manos Koukoutos
+Adam Klein
+Alexander Syrotenko
+Daniel Ehrenberg
+Ryan Hunt
+
+### Discussion: Observations about Types in Wasm (Andreas Rossberg) [1 hr]
```

Comment:
@rossberg Could you add your slides when you get a chance?

Reply:
@tlively, done in #666 (he he).